.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,55 @@
+
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Alternatives Considered
+      description: Describe any alternative solutions you've considered
+
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional Context
+      description: Add any other context or screenshots about the feature request here
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)